pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -2518,17 +2528,26 @@ public boolean isSizeBacklogExceeded() {
      * @return determine if backlog quota enforcement needs to be done for topic based on time limit
      */
     public boolean isTimeBacklogExceeded() {
+        try {
+            return checkTimeBacklogExceeded().get();
+        } catch (Throwable e) {
```

Comment:
This is a code smell

Reply:
This is just for keeping the same behavior of `isTimeBacklogExceeded` as before. This method is called somewhere else.

Reply:
I get it. Thx.
I will remove this `isTimeBacklogExceeded`.